src/client.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * A Mastodon toot object, with just the fields of toots that we need.
+ */
+export interface Toot {
```

Comment:
@sampsyo  BTW the `Toot` terminology was [officially replaced](https://gizmodo.com.au/2022/11/mastodon-has-officially-retired-the-toot-its-version-of-the-tweet/) some time ago:

> “Probably not a lot of people know this now, but Mastodon’s web app started out with ‘Publish’. In 2016, a famous YouTuber jokingly offered to support Mastodon’s Patreon forever if I changed the button to say ‘Toot’ instead. Needless to say, this was a really early, and not very informed decision. The first glimpse of attention and financial support. As a non-native speaker I had no idea there was another meaning,” Rochko said in a post last May. 

> “Developing Mastodon, I strive to use terminology that is familiar to as many people as possible so as not to put up unnecessary barriers in understanding. For this reason I am happy that we went away from calling posts ‘toots’. It has always been a point of friction for people,” he said, adding that using the word had a “negative effect” and offered little benefit. 

They now call them "messages" or "posts."

Reply:
Indeed! I knew about this but thought it was harmless to use the old name internally, as long as it never appears in the UI. I hope it doesn't come across as too unprofessional; just a little nod to the old days on Mastodon.

Reply:
For GitHub consumers of this NPM package, it would be publicly visible in the source code. In our case that code is `Copyright (c) Microsoft Corporation.` so maybe it could matter? It's [hard to say](https://www.onmsft.com/news/santagate-gets-github-all-riled-up-over-visual-studio-code-icon/).😅